lib/main_common.pm

```diff
@@ -1717,12 +1717,12 @@ sub load_common_x11 {
     }
 }
 
-# Move fips testsuites to main_common to apply to SLE_FIPS + openSUSE
-# Rename load_fips_tests_* to load_security_tests_* to avoid confusedness since
+# The function name load_security_tests_* is to avoid confusing since
 # openSUSE does NOT have FIPS mode
-# Some tests are only valid for SLE FIPS and loaded if defined variables set
+# Some tests are valid only for FIPS Regression testing. Use
+# "FIPS_ENABLED" to control whether to run these "FIPS only" cases
 sub load_security_tests_core {
-    if (check_var('DISTRI', 'sle') && get_var('FIPS_TS')) {
+    if (check_var('DISTRI', 'sle') && defined get_var('FIPS_ENABLED')) {
```

Comment:
Not sure why we need check if value is defined, as `undef` is false. But now in case variable is set to 0, condition will be true, which is confusing. WDYT? Otherwise LGTM.

Reply:
Yes, you are right. Thanks. I have changed.